examples/experimental/tutorial_dbt_dagster_v2/tutorial_dbt_dagster_v2/__init__.py

```diff
@@ -13,7 +13,7 @@
     resources={
         "dbt": DbtCli(
             project_dir=DBT_PROJECT_DIR,
-            global_config=["--no-use-colors"],
+            global_flags=["--no-use-colors"],
```

Comment:
Perhaps `global_config_flags`?
```suggestion
            global_config_flags=["--no-use-colors"],
```